### DISCUSSION TITLE
React doc viewer problem with ms files

### DISCUSSION THREAD

**Discussion by MohitRaykwar7389966852** [npm/feedback#899] [General]
I'm using react doc viewer of npm library to display the files it's ok with all file format but not working with Microsoft file I think we can do this with uploaded file that is publicly accessible but not with local files there is a option in this library to display local file to but it's not working with Microsoft local file so what can we do with this problem and how can we solve this if there is any solution of it please share

**Comment by ljharb** [comment]
If you have an issue with a specific npm package, you'll need to file it with that package, not with npm.
